twtr/grails-app/domain/twtr/Message.groovy

```diff
@@ -1,7 +1,7 @@
 package twtr
 
 class Message {
-    Account sentFromAccount
+    static belongsTo = Account
```

Comment:
Does the Message need to be back referenced to Account (i.e. create a column in the Message table with the Account Id)?   I think it should. If so,  the line needs to be: static belongsTo = [account: Account]. 
https://grails.github.io/grails-doc/3.0.x/ref/Domain%20Classes/belongsTo.html